content/en/tracing/setup_overview/setup/php.md

```diff
@@ -451,6 +451,24 @@ If no core dump was generated, check the following configurations and change the
 1. Ensure you have a suitable `ulimit` set in your system. You can set it to unlimited: `ulimit -c unlimited`.
 1. If your application runs in a Docker container, changes to `/proc/sys/*` have to be done to the host machine. Contact your system administrator to know the options available to you. If you are able to, try recreating the issue in your testing or staging environments.
 
+### Valgrind trace
+
+Preferably, if you want to give us as many hints as possible, try running the application with valgrind (install it from your package manager). This approach will also work in an unprivileged container (unlike core dumps by default).
```

Comment:
```suggestion
To gain more details about the crash, run the application with Valgrind. Unlike default core dumps, this approach works in an unprivileged container.

Install Valgrind with your package manager. Run the application with Valgrind enough to generate a few requests (not generally in production).
```

Reply:
a "default core dump" feels wrong to read, would then just write _Unlike core dumps_.